programs/voter-stake-registry/src/state/registrar.rs

```diff
@@ -57,6 +31,28 @@ impl Registrar {
             .position(|r| r.mint == mint)
             .ok_or(Error::ErrorCode(ErrorCode::VotingMintNotFound))
     }
+
+    pub fn max_vote_weight(&self, mint_accounts: &[AccountInfo]) -> Result<u64> {
+        self.voting_mints
+            .iter()
+            .try_fold(0u64, |mut sum, voting_mint_config| -> Result<u64> {
+                if !voting_mint_config.in_use() {
+                    return Ok(sum);
+                }
+                let mint_account = mint_accounts
```

Comment:
what if mint_accounts is empty?

Reply:
If it's empty `find()` returns `None` and a `VotingMintNotFound` error is returned.